CHANGELOG.md

```diff
@@ -47,6 +48,113 @@ Notable changes in reversed alphabetical order:
 
 The full list of included changes:
 
+- Backports ([#8624](https://github.com/paritytech/parity/pull/8624))
+  - Trace precompiled contracts when the transfer value is not zero ([#8486](https://github.com/paritytech/parity/pull/8486))
+    - Trace precompiled contracts when the transfer value is not zero
```

Comment:
This line is duplicated

Reply:
Coincidence: Title of PR vs first commit message.